faq/licensing.md

```diff
@@ -6,7 +6,7 @@ description: This article outlines the most frequently asked questions about Sys
 
 ## How is Syskit Point licensed?
 
-Syskit point price is calculated based on the number of licensed Microsoft 365 users in your tenant. You can choose between predefined editions according to your needs. Check out the [feature comparison](https://www.syskit.com/products/point/pricing/).
+The price for Syskit Point is calculated based on the number of licensed Microsoft 365 users in your tenant. Syskit Point does not count any licenses that have FREE in their name when determining price. You can choose between predefined editions according to your needs. Check out the [feature comparison](https://www.syskit.com/products/point/pricing/).
```

Comment:
prijedlog: You can choose between available Syskit Point plans according to your needs.
da si netko ne bi protumačio da govorimo da je moguće birati između editionima Microsoft licenci